src/javascripts/Views/referencedByView.js

```diff
@@ -0,0 +1,104 @@
+ngapp.controller('referencedByViewController', function($scope, $element, $timeout, htmlHelpers, treeService, referencedByViewService, referencedByViewColumnsService, hotkeyService, nodeSelectionService, treeViewService, treeColumnService, contextMenuService, contextMenuFactory) {
+    // link view to scope
+    $scope.view = $scope.$parent.tab;
+    $scope.view.scope = $scope;
+
+    $scope.allColumns = referencedByViewColumnsService.columns;
+    $scope.contextMenuItems = contextMenuFactory.referencedByViewItems;
+
+    // inherited functions
+    treeService.buildFunctions($scope, $element);
+    treeViewService.buildFunctions($scope, $element);
+    referencedByViewService.buildFunctions($scope);
+    treeColumnService.buildFunctions($scope, '.referenced-by-view', true);
+    nodeSelectionService.buildFunctions($scope, false);
+    hotkeyService.buildOnKeyDown($scope, 'onTreeKeyDown', 'referencedByView');
+
+    $scope.open = function(node) {
+        const record = node.handle ? node.handle : node;
+        $scope.record = record;
+        $scope.syncWithLinkedViews(record);
+    };
+
+    // scope functions
+    $scope.showContextMenu = function(e) {
+        contextMenuService.showContextMenu($scope, e);
+    };
+
+    $scope.toggleAddressBar = function(visible) {
+        $scope.showAddressBar = visible;
+        if (visible) {
+            $timeout($scope.focusAddressInput, 50);
+        } else if ($scope.treeElement) {
+            $scope.treeElement.focus();
+        }
+    };
+
+    $scope.focusAddressInput = function () {
+        let addressInput = htmlHelpers.resolveElement($scope.tabView, 'record-address-bar/input');
+        if (addressInput) addressInput.focus();
+    };
+
+    $scope.openColumnsModal = function() {
+        $scope.$emit('openModal', 'editColumns', {
+            allColumns: $scope.allColumns,
+            columnsService: referencedByViewColumnsService
+        });
+    };
+
+    $scope.onScroll = function(e) {
```

Comment:
this code was used on the record view because the columns on the record view may often exceed the width of the record view because it displays override records horizontally.  This code shouldn't be necessary for the referenced by view, which should display columns in a similar fashion to the tree view (fixed width, no horizontal scroll allowed).

I do notice that xEdit supports horizontal scroll on both the referenced by view and the tree view, but if we're going to add that we should do it as a separate PR so we can be consistent.  If the referenced by view is going to support horizontal scroll the tree view should as well.  (the logic related to their columns will have to change a bit, it's too big a change to do in this PR)